src/plugins/unified_search/public/query_string_input/query_bar_menu.tsx

```diff
@@ -88,15 +98,17 @@ export function QueryBarMenu({
   });
 
   const button = (
-    <EuiButtonIcon
-      size="m"
-      display="base"
-      onClick={onButtonClick}
-      iconType="filter"
-      aria-label={buttonLabel}
-      title={buttonLabel}
-      data-test-subj="showQueryBarMenu"
-    />
+    <EuiToolTip delay="long" content={buttonLabel}>
+      <EuiButtonIcon
```

Comment:
And I added this one too:
<img width="164" alt="Screen Shot 2022-04-12 at 15 38 17 PM" src="https://user-images.githubusercontent.com/549577/163066873-76f4f038-093b-4508-9886-fbfdf8dae1e7.png">